services/movie_session.py

```diff
@@ -42,3 +44,12 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> List[Dict[str, int]]:
+    taken_seats = Ticket.objects.filter(
+        movie_session_id=movie_session_id
+    ).values(
+        "row", "seat"
+    )
+    return [{"row": seat["row"], "seat": seat["seat"]} for seat in taken_seats]
```

Comment:
could u just make like this here 
list(taken_seats), why do u need dict comprehension here ?